impl/src/test/java/org/ehcache/docs/plugs/CharSequenceSerializer.java

```diff
@@ -39,18 +43,53 @@ public CharSequenceSerializer(ClassLoader classLoader) {
   @Override
   public ByteBuffer serialize(CharSequence object) {
     LOG.info("serializing {}", object);
-    ByteBuffer byteBuffer = ByteBuffer.allocate(object.length());
+
+    byte type;
+    if (object.getClass().equals(String.class)) {
```

Comment:
Please, can we _not_ demonstrate this? I would consider is fishy code if found in my code base, so why advertise it as an example.